src/js/components/Select/stories/Disabled.js

```diff
@@ -1,26 +1,27 @@
 import React, { useState } from 'react';
 
-import { Box, Select, Text } from 'grommet';
+import { Box, Grommet, Select, Text } from 'grommet';
 
 export const Disabled = () => {
   const options = ['one', 'two'];
   const [value, setValue] = useState('');
   return (
-    // Uncomment <Grommet> lines when using outside of storybook
-    // <Grommet theme={...}>
-    <Box fill align="center" justify="start" pad="large" gap="medium">
-      <Text weight="bold">Disabled</Text>
-      <Select
-        id="select"
-        name="select"
-        placeholder="Select"
-        value={value}
-        options={options}
-        onChange={({ option }) => setValue(option)}
-        disabled
-      />
-    </Box>
-    // </Grommet>
+    <Grommet
```

Comment:
If this is a custom themed story, it should be under a "custom theme" folder similar to: https://storybook.grommet.io/?path=/story/controls-accordion-custom-themed-custom--custom